vnext/ReactUWP/Views/module.g.cpp

```diff
@@ -1,19 +1,20 @@
-﻿// WARNING: Please don't edit this file. It was generated by C++/WinRT v1.0.190111.3
+// WARNING: Please don't edit this file. It was generated by C++/WinRT v1.0.190111.3
```

Comment:
This is not true. We should just delete this line.

Reply:
```suggestion
```